README.md

```diff
@@ -32,21 +32,21 @@ The following example shows the container used for a deployment step with a Azur
-| 2.42.0   | [Azure CLI 2.42.0](https://learn.microsoft.com/en-us/cli/azure/release-notes-azure-cli#november-01-2022) | swissgrc/azure-pipelines-dotnet:6.0.402   | 2.42.0    | ![Docker Image Size (tag)](https://img.shields.io/docker/image-size/swissgrc/azure-pipelines-azurecli/2.42.0?style=flat-square)   |
-| 2.43.0   | [Azure CLI 2.43.0](https://learn.microsoft.com/en-us/cli/azure/release-notes-azure-cli#december-06-2022) | swissgrc/azure-pipelines-dotnet:6.0.404   | 2.43.0    | ![Docker Image Size (tag)](https://img.shields.io/docker/image-size/swissgrc/azure-pipelines-azurecli/2.43.0?style=flat-square)   |
-| 2.44.0   | [Azure CLI 2.44.0](https://learn.microsoft.com/en-us/cli/azure/release-notes-azure-cli#january-10-2023)  | swissgrc/azure-pipelines-dotnet:6.0.405   | 2.44.0    | ![Docker Image Size (tag)](https://img.shields.io/docker/image-size/swissgrc/azure-pipelines-azurecli/2.44.0?style=flat-square)   |
-| 2.45.0   | [Azure CLI 2.45.0](https://learn.microsoft.com/en-us/cli/azure/release-notes-azure-cli#february-07-2023) | swissgrc/azure-pipelines-dotnet:6.0.406   | 2.45.0    | ![Docker Image Size (tag)](https://img.shields.io/docker/image-size/swissgrc/azure-pipelines-azurecli/2.45.0?style=flat-square)   |
-| 2.46.0   | [Azure CLI 2.46.0](https://learn.microsoft.com/en-us/cli/azure/release-notes-azure-cli#march-07-2023)    | swissgrc/azure-pipelines-dotnet:6.0.407   | 2.46.0    | ![Docker Image Size (tag)](https://img.shields.io/docker/image-size/swissgrc/azure-pipelines-azurecli/2.46.0?style=flat-square)   |
+| Tag      | Description                                                                                              | Base Image                                      | Azure CLI | Size                                                                                                                              |
+|----------|----------------------------------------------------------------------------------------------------------|-------------------------------------------------|-----------|-----------------------------------------------------------------------------------------------------------------------------------|
+| latest   | Latest stable release (from `main` branch)                                                               | swissgrc/azure-pipelines-dotnet:6.0.407         | 2.46.0    | ![Docker Image Size (tag)](https://img.shields.io/docker/image-size/swissgrc/azure-pipelines-azurecli/latest?style=flat-square)   |
+| unstable | Latest unstable release (from `develop` branch)                                                          | ghcr.io/swissgrc/azure-pipelines-dotnet:6.0.407 | 2.46.0    | ![Docker Image Size (tag)](https://img.shields.io/docker/image-size/swissgrc/azure-pipelines-azurecli/unstable?style=flat-square) |
```

Comment:
We shouldn't change image name to a specific repository, since we publish to multiple registries